src/views/zesty/ForDeveloper.js

```diff
@@ -59,28 +59,191 @@
+    ...pageProps,
+  };
+
+  const zestyDrivesData =
+    content.numbers?.data.reduce((acc, item) => {
+      acc.push({
+        feature_name: item.feature_name,
+        content: item.content,
+      });
+
+      return acc;
+    }, []) || [];
+
+  const zestyDrivesProps = {
+    header_size: 32,
+    textHighlight: '',
+    card_name_color: theme.palette.zesty.zestyZambezi,
+    data: zestyDrivesData,
+    features_header: content.numbers_title,
+    ...pageProps,
+  };
+
+  const integrationsProps = {
+    text_content: content.integrations_text,
+    logos: content.integrations_graphic?.data[0]?.url,
+    cta_button_text: content.integrations_button_text || FillerContent.cta,
+    cta_button_link: content.integrations_button_link || FillerContent.href,
+    ...pageProps,
+  };
+
+  const supportProps = {
+    eyebrow: content.support_eyebrow || FillerContent.header,
+    titleAndDescription: content.support_text || FillerContent.rich_text,
+    mainImage:
+      content.nextjs_graphic?.data[0]?.url || FillerContent.photos[0].src,
+    primaryCta: content.footer_button_2 || FillerContent.cta,
+    primaryCtaLink:
+      content.footer_button_2_link.data[0].meta.web.uri || FillerContent.href,
+    ...pageProps,
+  };
+
+  const documentationProps = {
+    header: content.docs_text || FillerContent.header,
+    documentations: content.dev_docs || FillerContent.rich_text,
+    ...pageProps,
+  };
+
+  const testimonialsProps = {
+    title: content.testimonials_title,
+    data: content.testimonials?.data,
+  };
+
+  const bottomProps = {
+    graphic: content?.footer_cta_graphic?.data[0].url || '',
+    titleAndDescription: content.footer_cta_text || FillerContent.rich_text,
+    cta_text: content.footer_button_1 || FillerContent.cta,
+    secondary_cta_text: content.footer_button_2 || FillerContent.cta,
+    secondary_cta_link:
+      content.footer_button_2_link?.data[0].meta.web.uri || FillerContent.href,
+    ...pageProps,
+  };
+
+  const howZestyWorksProps = {
+    header: content.zesty_for_the_team || FillerContent.description,
+    teamLinks: content.zesty_for_the_team_links || FillerContent.rich_text,
+    ...pageProps,
+  };
+
   return (
     <>
-      {/* Zesty.io Output Example and accessible JSON object for this component. Delete or comment out when needed.  */}
-      <h1
-        dangerouslySetInnerHTML={{ __html: content.meta.web.seo_meta_title }}
-      ></h1>
-      <div>{content.meta.web.seo_meta_description}</div>
-      <div
-        style={{
-          background: '#eee',
-          border: '1px #000 solid',
-          margin: '10px',
-          padding: '20px',
-        }}
-      >
-        <h2>Accessible Zesty.io JSON Object</h2>
-        <pre>{JSON.stringify(content, null, 2)}</pre>
-      </div>
-      {/* End of Zesty.io output example */}
+      <Hero {...heroProps} />
+      <ContainerWithBackground {...nextJsProps} />
+      <WhyZesty {...whyZestyProps} />
```

Comment:
@jeromeZestyIO 

Use the feature components from /src/blocks/features/Features/Features.js